container/ubuntu16_04/layers/bazel/BUILD

```diff
@@ -26,22 +29,39 @@ language_tool_layer(
     name = "bazel-tools",
     base = "@ubuntu16_04//image",
     packages = [
+        "bash-completion",
+        "binutils",
         "patch",
         "unzip",
         "zip",
     ],
 )
 
-language_tool_layer(
-    name = "bazel-ltl",
-    additional_repos = [
-        "deb [arch=amd64] http://storage.googleapis.com/bazel-apt stable jdk1.8",
-    ],
+toolchain_container(
+    name = "bazel_installer_image",
     base = "@ubuntu16_04//image",
-    keys = [
-        "@bazel_gpg//file",
+    files = [
+        "@bazel_installer//file",
     ],
     packages = [
-        "bazel",
+        "unzip",
+    ],
+)
+
+container_run_and_extract(
+    name = "bazel_tar",
+    commands = [
+        "cd /",
+        "chmod +x bazel-" + BAZEL_VERSION + "-installer-linux-x86_64.sh",
+        "./bazel-" + BAZEL_VERSION + "-installer-linux-x86_64.sh",
+        "tar cvf bazel.tar /usr/local/bin/bazel /usr/local/lib/bazel/*",
```

Comment:
Is it possible that we remove the timestamp? Something like:
https://github.com/bazelbuild/bazel-toolchains/blob/20ea2c76b02d0c20fe95effffcd433b76909cf0b/skylib/container_file_export.sh#L42

Reply:
done